lib/transformer.dart

```diff
@@ -85,6 +100,47 @@ class InitializeTransformer extends Transformer {
       }
     });
   }
+
+  Future _replaceEntryWithBootstrap(Transform transform) {
+    // For now at least, _htmlEntryPoint, _entryPoint, and _newEntryPoint need
```

Comment:
On second thought I don't think we really want people reaching into web from test, etc.